tfjs-backend-webgpu/src/kernels/reduce_webgpu.ts

```diff
@@ -78,7 +76,7 @@ export class ReduceProgram implements WebGPUProgram {
     }
 
     const outputSnippet = this.reduceType === 'mean' ?
-        `setOutput(flatOutputIndex, bestValue / ${this.reduceSize});` :
```

Comment:
This is the place where we forgot to use uniform instead of the constant value.  Or this value should be as part of the shader key. Otherwise, the shader key can correctly reflect different shader variants.